examples/solvers/chambolle_pock_deconvolve.py

```diff
@@ -56,80 +56,37 @@
 from __future__ import print_function, division, absolute_import
 from future import standard_library
 standard_library.install_aliases()
-from builtins import super
 
 import numpy as np
-import scipy
-import scipy.ndimage
 import odl
 
-
-# Define the forward operator of the inverse problem in question
-class Convolution(odl.Operator):
-    def __init__(self, space, kernel, adjkernel):
-        self.kernel = kernel
-        self.adjkernel = adjkernel
-        super().__init__(space, space, linear=True)
-
-    def _call(self, rhs, out):
-        scipy.ndimage.convolve(rhs, self.kernel, output=out.asarray(),
-                               mode='wrap')
-        return out
-
-    @property
-    def adjoint(self):
-        return Convolution(self.domain, self.adjkernel, self.kernel)
-
-
-# Gaussian kernel for the convolution operator
-def kernel(x):
-    mean = [0.1, 0.5]
-    std = [0.1, 0.1]
-    k = np.exp(-(((x[0] - mean[0]) / std[0]) ** 2 +
-                 ((x[1] - mean[1]) / std[1]) ** 2))
-    return k
-
-
-# Kernel for the adjoint of the convolution operator
-def adjkernel(x):
-    return kernel((-x[0], -x[1]))
-
-
-# Continuous definition of problem
-cont_space = odl.FunctionSpace(odl.Rectangle([-1, -1], [1, 1]))
-
-# Create a kernel space which has twice the extent of the image space
-kernel_space = odl.FunctionSpace(cont_space.domain - cont_space.domain)
-
 # Discretization parameters
 n = 50
-npoints = np.array([n + 1, n + 1])
-npoints_kernel = np.array([2 * n + 1, 2 * n + 1])
 
 # Discretized spaces
-discr_space = odl.uniform_discr_fromspace(cont_space, npoints)
-discr_kernel_space = odl.uniform_discr_fromspace(kernel_space, npoints_kernel)
-
-# Discretize the functions
-disc_kernel = discr_kernel_space.element(kernel)
-disc_adjkernel = discr_kernel_space.element(adjkernel)
+space = odl.uniform_discr([0, 0], [n, n], [n, n])
 
-# Initialize convolution operator
-convolution = Convolution(discr_space, disc_kernel, disc_adjkernel)
+# Initialize convolution operator by fourier formula
+filter_width = 0.03 * n
+ft = odl.trafos.FourierTransform(space)
+C = filter_width**2 / 4.0**2  # by fourier transform of gaussian function
```

Comment:
Meaning of the comment?

Also, always capitalize "Fourier" and "Gaussian", they're (derived from) names of persons.